internal/bminventory/inventory.go

```diff
@@ -4451,42 +4444,12 @@ func (b *bareMetalInventory) setPullSecretFromOCP(cluster *common.Cluster, log l
 	return nil
 }
 
-func formatStaticIPs(staticIpsConfig []*models.StaticIPConfig) string {
-	lines := make([]string, len(staticIpsConfig))
-
-	// construct static IPs config to string
-	for i, entry := range staticIpsConfig {
-		elements := []string{entry.Mac}
-		elements = append(elements, getStaticIPsData(entry.IPV4Config)...)
-		elements = append(elements, getStaticIPsData((*models.StaticIPV4Config)(entry.IPV6Config))...)
-		lines[i] = strings.Join(elements, ";")
-	}
-
-	sort.Strings(lines)
-
-	return strings.Join(lines, "\n")
-}
-
-func formatVlans(vlansConfig []*models.VlanConfig) string {
-	lines := make([]string, len(vlansConfig))
-	// construct vlans config to string
-	for i, entry := range vlansConfig {
-		elements := []string{entry.Mac, strconv.FormatInt(entry.ID, 10)}
-		elements = append(elements, getStaticIPsData(entry.IPV4Config)...)
-		elements = append(elements, getStaticIPsData((*models.StaticIPV4Config)(entry.IPV6Config))...)
-		lines[i] = strings.Join(elements, ";")
-	}
+func formatStaticNetwork(staticNetworkConfig []string) string {
+	lines := make([]string, len(staticNetworkConfig))
 
+	copy(lines, staticNetworkConfig)
 	sort.Strings(lines)
-
-	return strings.Join(lines, "\n")
-}
-
-func getStaticIPsData(static_config *models.StaticIPV4Config) []string {
-	if static_config != nil {
-		return []string{static_config.IP, static_config.Mask, static_config.DNS, static_config.Gateway}
-	}
-	return []string{"", "", "", ""}
+	return strings.Join(lines, "ZZZZZ")
```

Comment:
what is `ZZZZZ`

Reply:
i needed some delimeter between hosts configuration. ZZZZ sounds as good as any

Reply:
then please add a comment explaining it.

Reply:
done